### PR TITLE
Make remaining python sources PEP-8 compliant

### PR DESCRIPTION
**Pull request by kamalesh-babulal**
This is the next batch of PEP-8 fixes. It introduces string quoting and
multiple lines, as in #36. The various PEP-8 warnings fixed across the
files are:

-   E111 indentation is not a multiple of 4
-   E117 over-indented
-   E122 continuation line missing indentation or outdented
-   E122 continuation line missing indentation or outdented
-   E122 continuation line missing indentation or outdented
-   E128 continuation line under-indented for visual indent
-   E261 at least two spaces before inline comment
-   E302 expected 2 blank lines, found 1
-   E303 too many blank lines (2)
-   E305 expected 2 blank lines after class or function definition, found 1
-   E501 line too long 
-   E502 the backslash is redundant between brackets
-   E712 comparison to False should be 'if cond is False:' or 'if not cond:'
-   E722 do not use bare 'except'
-   F401 '...' imported but unused
-   F841 local variable '...' is assigned to but never used

### CODE REVIEW

File: ftests/controller.py [hunk 1]
```diff
@@ -33,7 +34,8 @@ def __str__(self):
         out_str = "Controller {}\n".format(self.name)
```
[the remark this answers is not in the document]
Comment (kamalesh-babulal) [reply ftests/controller.py:34]: Thanks for the reviews @drakenclimber.  Ah !! I missed converting in this file.

File: ftests/container.py [hunk 2]
```diff
@@ -56,25 +55,27 @@ def __init__(self, name, stop_timeout=None, arch=None, cfg_path=None,
...
+        out_str += '\n\tstop_timeout = {}\n'.format(self.stop_timeout)
 
         return out_str
 
     # configure the container to meet our needs
     def config(self):
         ftest_dir = os.path.dirname(os.path.abspath(__file__))
         tests_dir = os.path.dirname(ftest_dir)
-        libcg_dir = os.path.dirname(tests_dir)
+        self.libcg_dir = os.path.dirname(tests_dir)
```
[the remark this answers is not in the document]
Comment (kamalesh-babulal) [reply ftests/container.py:71]: Make sense, will do that.

**Comment by kamalesh-babulal** [comment]
** V2 Changes**
- Addressed the review comment from @drakenclimber 
- Fixed other warnings in `process.py`
- Added `ftests.py` PEP-8 warning fixes (new commit)

**Comment by drakenclimber** [comment]
Merged.  Thanks, @kamalesh-babulal

**Pull request closed without merging**